test/OrchardCore.Tests/Apis/Context/TestRecipeHarvester.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Extensions.FileProviders;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Recipes.Services;
+using Xunit;
+
+namespace OrchardCore.Tests.Apis.Context
+{
+    public class TestRecipeHarvester : IRecipeHarvester
+    {
+        private readonly IRecipeReader _recipeReader;
+
+        public TestRecipeHarvester(IRecipeReader recipeReader)
+        {
+            _recipeReader = recipeReader;
+        }
+
+        public Task<IEnumerable<RecipeDescriptor>> HarvestRecipesAsync()
+            => HarvestRecipesAsync(new[]
+            {
+                "Apis/Lucene/Recipes/luceneQueryTest.json"
+            });
+
+        private async Task<IEnumerable<RecipeDescriptor>> HarvestRecipesAsync(string[] paths)
+        {
+            var recipeDescriptors = new List<RecipeDescriptor>();
+
```

Comment:
Please remove the extra line between the `var`

Reply:
Removed